README.md

```diff
@@ -104,12 +115,234 @@ Ansible based automation
 
 You can use the Ansible playbooks and roles at <https://github.com/rh-nfv-int/nfv-example-cnf-deploy> to automate the use of the Example CNF.
 
+Load balancer mode vs. direct mode
+------------------------
+
+There are two different ways of deploying example-cnf, depending on the `enable_lb` flag (default to `true`):
+
+- Load balancer mode (`enable_lb: true`):
+    - Represents the behavior described above, with a TestPMD instance acting as load balancer between TRex and the CNF Applications.
+- Direct mode (`enable_lb: false`):
+    - The TestPMD instance is not used, then testpmd-lb-operator is not deployed.
+    - In this case, there's a direct connection between TRex and CNF Application.
+    - The CNF Application only uses one replica pod, instead of two.
+
+Pod affinity rules
+------------------------
+
+There are different, possible affinity rules for the deployed pods, depending whether testpmd-lb-operator is deployed (load balancer mode) or not (direct mode):
+
+- Load balancer mode:
+    - The `loadbalancer-<x>` pod is deployed in a different worker than `trexconfig-<x>` and `testpmd-app-<x>`.
+    - There are no constraints for the location of `trexconfig-<x>` and `testpmd-app-<x>` pods, so they may live in the same worker.
+- Direct mode:
+    - Since load balancer is not used, in this scenario, `trexconfig-<x>` and `testpmd-app-<x>` pods are placed in different worker nodes.
```

Comment:
Right above we only discussed pods, and now jump into config which is pinned by CR but was not even mentionned before. We could probably add more details for each operator, adding some pictures or more detailed explanations. 

![image](https://github.com/openshift-kni/example-cnf/assets/26544656/eac7fd3f-1647-4de3-90d0-56339bc84f9f)

Reply:
I've included the consideration for CRs, I'll add also some diagrams. Regarding affinity rules, we need to talk about pods in that point since these are specs for the pods, I'll keep using this for the moment

Reply:
Since the diagram is in fact common for all operators, since it depicts the reconciliation loop, I'll add it to the section where we talk about operator-sdk